slang-gfx.h

```diff
@@ -1221,6 +1221,32 @@ class IRayTracingCommandEncoder : public ICommandEncoder
 class ICommandBuffer : public ISlangUnknown
 {
 public:
+    struct NativeBufferHandle
```

Comment:
I think we can just do `typedef uint64_t NativeHandle;` here and remove all the boilerplate.